threedi_scenario_downloader/downloader.py

```diff
@@ -814,10 +869,12 @@ def to_datetime_obj(time_string):
 
 
 def get_raster_timesteps(raster, interval_hours=None):
-    """returns a list of 'YYYY-MM-DDTHH:MM:SS' formatted timesteps in temporal range of raster object
+    """returns a list of '%Y-%m-%dT%H:%M:%SZ' formatted timesteps in temporal
```

Comment:
Op zich is zo'n formaat als `YYYY-MM-DD` nuttiger dan `%Y-%m-%d`, want ik moet elke keer als ik met zo'n formaat bezig ben weer kijken of `%m` nou een maand met/zonder nul-aan-het-begin is of dat het minuten zijn.